galasa-managers-parent/galasa-managers-comms-parent/dev.galasa.http.manager/src/main/java/dev/galasa/http/internal/HttpClientImpl.java

```diff
@@ -341,6 +344,21 @@ private void addHeaders(AbstractHttpMessage message, ContentType contentType, Co
 
     }
 
+    public void getFile(Path destination, String path) {
+        try{
+            HttpClientRequest request = HttpClientRequest.newGetRequest(buildUri(path, null).toString(),
+                new ContentType[] { ContentType.APPLICATION_OCTET_STREAM });
+
+            CloseableHttpResponse repsonse = execute(request.buildRequest());
+            OutputStream output = new FileOutputStream(destination.toFile());
```

Comment:
Nope,  sorry you got this wrong.    Files.getOuputstream(destination) I believe,  or something like that.   At the moment,  you are forcing it to be on the local filesystem.   using Files means it can be on ANY filesystem, including the RAS